app/components/dashboard/dashboard.component.js

```diff
@@ -263,12 +272,27 @@ class Dashboard extends React.Component {
                         })}
                         results={leaderboard}
                         value={searchString}
+                        placeholder={"Search Team Name"}
                       />
                     </div>
                     <div>
                       <ResizableListView height={160} className={"scrolling"} />
                     </div>
                   </div>
+                  <div className="content-container-leaderboard">
+                    <Header size="medium" className="container-header">
+                      <FormattedMessage id="dashboard.teamLeaderboard" />
+                    </Header>
+                    <TeamLeaderboardTable
+                      height={450}
+                      className={'scrolling'}
+                      teams={teams}
+                      data={breakdown.offices}
```

Comment:
Same as above, the TeamLeaderboardTable doesn't actually use data, list, prefix and image props so they can be removed?

Reply:
Refactored